README.md

```diff
@@ -62,52 +62,57 @@ For production use a database backed session store is recommended - such as [con
+* `translate` - provide a function for translating validation error codes into usable messages. Previous implementations have used [i18next](https://www.npmjs.com/package/i18next) to do translations.
+*  `templatePath` - provides the location within `app.get('views')` that templates are stored. Default `pages`.
+* `controller` - override the [default controller](./lib/controller.js) for steps without a controller specified.
+* `params` - define a suffix for the routes for supporting additional URL parameters.
 
 Remaining field options documentation can be found in the hmpo-template-mixins [README](https://github.com/UKHomeOffice/passports-template-mixins#options-1).
 
-### Additional wizard options
-
-A number of options can be passed to the wizard as a third argument to customise aspects of the behaviour for all steps.
-
-`translate` - provide a function for translating validation error codes into usable messages. Previous implementations have used [i18next](https://www.npmjs.com/package/i18next) to do translations.
-`templatePath` - provides the location within `app.get('views')` that templates are stored. Default `pages`.
-`controller` - override the [default controller](./lib/controller.js) for steps without a controller specified.
-`params` - define a suffix for the routes for supporting additional URL parameters.
-
 ### Custom Controllers
 
 Creating a custom controller:
 
 ```javascript
 // controller.js
-var util = require('util'),
-    Controller = require('hmpo-form-wizard').Controller;
-
-function CustomController() {
-  Controller.apply(this, arguments);
-  // extra middleware to log the request
-  this.use(function (req, res, next) {
-    console.log(req.method, req.url);
-    next();
-  });
-}
+const Controller = require('hmpo-form-wizard').Controller;
 
-util.inherits(CustomController)
+class CustomController extends Controller) {
+  middlewareSetup() {
+    super.middlewareSetup();
+
+    this.use((req, res, next) => {
+      console.log(req.method, req.url);
+      next();
+    });
+  }
+}
 
 module.exports = CustomController
 ```
+O
 
 Examples of custom controllers can be found in the [example app](./example/controllers)
 
 ## Example app
 
 An example application can be found in [the ./example directory](./example). To run this, follow the instructions in the [README](./example/README.md).
+
+## Migrating to wizard v6
+
+* The code has been updated to es6 and requires a minimum of Node v4
+* If addition middleware has been added by overriding the `constructor` or `requestHandler` method, this will need to be moved to one of the middleware setup methods (`middlewareSetup`, `middlewareChecks`, `middlewareActions`, or `middlewareLocals`) (see example above)
```

Comment:
Also that custom controller example could do something more useful with other bits of the life cycle